sdk/identity/identity/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- All of the credentials now include `allowPiiLogging` on their constructor options. If set to true, Personal Identifiable Information (PII) will be displayed in trace logs.
```

Comment:
Is `allowPiiLogging` the agreed upon name in the feature crew? Am just curious about the PII becoming Pii

Reply:
It hasn’t been decided yet. Here’s Vinay saying that this hasn’t been decided yet: https://github.com/Azure/azure-sdk-for-js/pull/17321#discussion_r699610389